.changeset/spotty-buckets-hang.md

```diff
@@ -0,0 +1,33 @@
+---
+"@biomejs/biome": major
+---
+
+Reworked some recommended rules recommended to be less pedantic and blocking. This is a **breaking change** if your project relied on those rules to block the CI in case of violations, you should raise their diagnostic to **error**.
```

Comment:
Will this be handled by `biome migrate`? I think it will be good to mention `biome migrate` in changesets such as these.

Reply:
We can definitely create a the migration, however I'm a bit afraid it won't work. Meaning, we can add it and it will work the first time. 

However, in the future, if the use decides to keep the recommended configuration, and they run `migrate`, this migration will kick in again. Maybe we should run this migration based on the version number

Reply:
> Maybe we should run this migration based on the version number.

This is the way :)